plugins/modules/win_disk_facts.py

```diff
@@ -18,6 +18,15 @@
     U(https://msdn.microsoft.com/en-us/library/windows/desktop/hh830612.aspx)
 author:
   - Marc Tschapek (@marqelme)
+options:
+  filter:
+    description:
+      - Allows to filter returned facts by type of disk information.
+      - If volumes are selected partitions will be returned as well.
+    type: list
+    elements: str
+    choices: [ physical_disk, virtual_disk, win32_disk_drive, partitions, volumes ]
+    default: [ physical_disk, virtual_disk, win32_disk_drive, partitions, volumes ]
```

Comment:
Needs `version_added: 1.9.0` to denote when this option was introduced in the collection.

Reply:
updated doc and added change fragment.